DESCRIPTION

```diff
@@ -33,3 +33,5 @@ Imports:
     tidyr,
     tidyselect,
     vegan
+Depends: 
+    R (>= 2.10)
```

Comment:
Isn't this a super wide range? I think we're at R v 4 something now. Is that a problem?

Reply:
it IS a super wide range...these depends/imports are autogenerated by `devtools::use_package()`. I've been following the R package guide to a T so I think it's ok. Prior to my adding data, it didn't depend on any specific version of R, so I think something about the data format wasn't implemented until 2.10